sig-instrumentation/annual-report-2021.md

```diff
@@ -0,0 +1,77 @@
+# SIG Instrumentation Annual Report 2021
+
+This report reflects back on CY 2020 and was written in Feb-Mar. 2021.
+
+## Operational
+
+*   How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+    *   Is your README accurate? have a CONTRIBUTING.md file?
+        *   Yes. We do not require a separate CONTRIBUTING.md file.
+    *   All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sig-list.md)?
+        *   Yes.
+    *   What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+        *   We have lively biweekly regular meetings and biweekly triage sessions. Our recordings are relatively up to date and we receive requests for them so we believe they are being watched. Our community archives are all up to date.
+*   How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+    *   We request these in SIG meetings and asynchronously for KubeCon and other community updates.
+    *   See above for OWNERS updates.
```

Comment:
I'm probably missing something here...can you point me to the process for keeping OWNERS up to date?

Reply:
Ah, there were a bunch of scratch notes that didn't make it in here. This was referring to https://github.com/kubernetes/community/pull/5603/files#diff-c7b01de0190da099e3c4819f213c78276bb6514a43f9f51339277f65c7ee7b29R10-R11

When we went through reviewing our charter and subprojects, we also reviewed all our OWNERS files, which are mainly in subprojects as we only own a small portion of k/k.

Reply:
The updates were https://github.com/kubernetes/kubernetes/pull/99013 and https://github.com/kubernetes/org/pull/2497 https://github.com/kubernetes/org/pull/2541